schedule/yast2_gui.yaml

```diff
@@ -6,13 +6,8 @@ description:    >
     as the gnome environment for the GUI tests.
 vars:
     BOOTFROM: c
-    BOOT_HDD_IMAGE: 1
-    # DESKTOP: gnome -> When added, an empty key appears first in the list
     HDDSIZEGB: 20
-    # HDD_1: SLES-%VERSION%-%ARCH%-Build%BUILD%@%MACHINE%-gnome.qcow2 -> if added, openQA will not publish in UI
     SOFTFAIL_BSC1063638: 1
-    START_AFTER_TEST: create_hdd_gnome
```

Comment:
I'm not sure if this really works. We should test cloning create_hdd_gnome and its child and see properly displayed the dependency tab.

Reply:
This variable is specified in Test Suite settings also. So, they are duplicated. If we decide to use them in yaml files, then let's remove them from the openQA. 

Also, I'm not sure if it is used by scheduler. So that, we should identify the list of variables that are used by scheduler and cannot be reached from yaml, and just do not include them here.

Reply:
It is documented that it doesn't work, so it is ok to remove it.